src/js/components/UserComponent.js

```diff
@@ -137,8 +135,7 @@ module.exports = React.createBackboneClass({
 
   render() {
     const key = utils.campusKey(this.getModel());
-    this.getModel().get('courses').comparator = this.getModel().get('courses').reverse;
-    this.getModel().get('courses').sort();
+    this.getModel().get('courses').models.sort((a, b) => a.startDate < b.startDate ? -1 : 1)
```

Comment:
Did you verify this works? It seems like it would just return a new array. Maybe it makes more since to go on the next line?

```js
const courses = this.getModel().get('courses').sort((a, b) => a.startDate < b.startDate ? -1 : 1).map(//...
```

Reply:
I did, but I did a more thorough test just now by making 4 classes vs just 2 in case they were ordering correctly just by chance.  They are ordering correctly still, but they also do when the line is commented out.  Your solution works too, but console.logging showed .startDate was returning undefined.  But I found how to access start_date.  It was pretty nested.